site/js/index.js

```diff
@@ -0,0 +1,14 @@
+// import utils
```

Comment:
For what it's worth, it would be nice if `api` were imported and published (i.e. `window.api === require('./util/api')`), since it makes debugging/testing things quite a bit easier :)

Reply:
I'll fix this now.

Reply:
Done!